commands/sandbox.go

```diff
@@ -562,6 +599,16 @@ func getSandboxDirectory() (string, bool) {
 	return sandboxDir, exists
 }
 
+// getCredentialDirectory returns the directory in which credentials should be stored for a given
+// CmdConfig.  The actual leaf directory is a function of the access token being used.  This ties
+// sandbox credentials to DO credentials
+func getCredentialDirectory(c *CmdConfig, sandboxDir string) string {
+	token := c.getContextAccessToken()
+	runes := []rune(token)
+	leafDir := string(runes[:8])
```

Comment:
The DigitalOcean API token format is currently being transitioned. Soon all new tokens will contain an identifiable, versionized prefix and match this regex: `^dop_v\d_([0-9a-f]){64}$`. So 8 characters might not be enough here. Wonder if we can just use a hash?

Reply:
Certainly.   That will guarantee uniqueness regardless of token format.   I will make that change.

Reply:
Done.